### PR TITLE
update accuracy evaluation scripts

### PR DESCRIPTION
**Pull request by WanliZhong**
Some information about models or evaluation scripts are outdated. This PR aims at updating them. Merged after #185

### CODE REVIEW

File: tools/eval/datasets/imagenet.py [hunk 1]
```diff
@@ -63,3 +64,1004 @@ def get_result(self):
     def print_result(self):
         print("Top-1 Accuracy: {:.2f}%; Top-5 Accuracy: {:.2f}%".format(self.top1_acc*100, self.top5_acc*100))
 
+
+    LABELS_IMAGENET_1K = '''tench
```
Comment (fengyuentau) [inline tools/eval/datasets/imagenet.py:68]: We dont need these labels for evaluation.

Comment (WanliZhong) [reply tools/eval/datasets/imagenet.py:68]: But the `model.infer(img)` will give labels directly not index of labels.

Comment (fengyuentau) [reply tools/eval/datasets/imagenet.py:68]: I suggest to add `load_label=True` in for example `PPResNet` then set it `False` in evaluation.

Comment (WanliZhong) [reply tools/eval/datasets/imagenet.py:68]: I will create another PR to modified it.

Comment (WanliZhong) [reply tools/eval/datasets/imagenet.py:68]: What's the difference between `text_recognition_CRNN_EN_2021sep.onnx` and `text_recognition_CRNN_EN_2023feb.onnx`? They have the same accuracy through evaluation.

**Review by fengyuentau — APPROVED** [review]
LGTM 👍

**Pull request merged**